### PR TITLE
added comands to update submodules

### PR DESCRIPTION
**Pull request by mandric**
Caveat here is this only works with the `branch = .` tweak in .gitmodules, which has been added to 2.12.x and master.

### CODE REVIEW

File: development/releasing.md [hunk 1]
```diff
@@ -10,7 +10,14 @@ When all the required issues are fixed in master it's time to put together a rel
 ```bash
 npm install --save libphonenumber@latest
 ```
-  - Update the git submodules (api and sentinel) to the latest and commit to `master` and push.
+  - Confirm the the git submodules (api and sentinel) are on the latest commit on the right branch (should mirror the parent repo).
+```bash
+git submodule update --remote api
+git submodule update --remote sentinel
+git status --exit-code api sentinel || \
+git commit -m 'bumping submodules' api sentinel && \
```
Comment (alxndrsn) [inline development/releasing.md]: interesting, didn't know you could include filenames here.  Would it also work:
```
git commit -m 'bumping submodules' -- api sentinel
```
?

File: development/releasing.md [hunk 2]
```diff
@@ -10,7 +10,14 @@ When all the required issues are fixed in master it's time to put together a rel
...
+git push
```
Comment (alxndrsn) [inline development/releasing.md]: After this push, is there a step to wait to see that the build still passes?

Comment (mandric) [inline development/releasing.md]: Yeah will add that.

File: development/releasing.md (hunk 1 above)
Comment (mandric) [inline development/releasing.md]: I don't know what the `--` is for.

**Comment by mandric** [comment]
I'll also add the `.gimodules` tweak to the active branches...

**Comment by mandric** [comment]
Ok, updated remaining release branches (2.11.x, 2.10.x, 2.8.x and 0.4.x) with .gitmodules tweak.

**Comment by mandric** [comment]
Now we have a `bump-submodules` script... just not sure where to put it.

**Comment by mandric** [comment]
Output looks like:
```
mandric in ~/dev/medic-webapp (0.4.x)
$ /tmp/bump-submodules 
diff --git a/api b/api
index 72cb12b8..f314730f 160000
--- a/api
+++ b/api
@@ -1 +1 @@
-Subproject commit 72cb12b84904efcaac40fc9910919aaf4ee0ba48
+Subproject commit f314730fb4a8e0f68d05824b571d4bb8b1a1767c
[0.4.x 9ddbf712] bumping submodules
 1 file changed, 1 insertion(+), 1 deletion(-)
Counting objects: 2, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (2/2), done.
Writing objects: 100% (2/2), 256 bytes | 0 bytes/s, done.
Total 2 (delta 1), reused 0 (delta 0)
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
To github.com:medic/medic-webapp.git
   656161d0..9ddbf712  0.4.x -> 0.4.x
Submodules updated and pushed, confirm new build in CI succeeds.
```

**Pull request merged**